### ISSUE TITLE
Set cardinality of ReplicationControllers -> Pods as (One -> Many) not (Many <-> Many)

### ISSUE THREAD

**Issue by jayunit100** [GoogleCloudPlatform/kubernetes#11981]
From https://groups.google.com/forum/#!topic/google-containers/-pA2pIsM8yU...  its possible that RC <-> Pod can be a many-many relationship, because RC's don't mark their territory !

This means RC's can fight over pods.

You can see the fallout of this in the GetPodControllers function in listers.go, where you have to return a list of replicationControllers managing a pod.  

This is both counterintuitive and also can cause some bugs.

**Comment by jayunit100** [comment]
cc @timothysc @rrati @lavalamp .  This may/may not effect HA haven't thought it out entirely.

**Comment by lavalamp** [comment]
What do you mean by "fix"? There's probably already issues about this, btw.

**Comment by jayunit100** [comment]
Happy to close if another issue is out there.... I didnt see one though?

**Comment by mikedanese** [comment]
We currently consider this to be user error. Results are deterministic for RC overlap after #10443 but the problem will be exacerbated when we add more pod controllers. I was looking for an "Ownership" issue but I couldn't find one.

**Comment by lavalamp** [comment]
Hm, I can't find one either, I guess I'm thinking of various PRs. This is one of @bprashanth's favorite issues. ;)

See #10443 and #10524; also #2395 and #10477 for PRs we didn't merge.

**Comment by lavalamp** [comment]
Oh, and I think this is completely orthogonal to HA.

**Comment by yujuhong** [comment]
I believe #2210 is the original issue

**Comment by jayunit100** [comment]
Okay perfect.  I'll close this guy and follow up in #2210 !
